api-reference/_hidden/dxDataGridColumn/editCellTemplate.md

```diff
@@ -69,18 +69,26 @@ Use the **editCellTemplate** to replace the default editor. In the template, spe
 
 Other properties that allow editor customization include:
 
-- **columns[]**.[editorOptions](/api-reference/_hidden/GridBaseColumn/editorOptions.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#editorOptions')         
+- **columns[]**.[editorOptions]({basewidgetpath}/Configuration/columns/#editorOptions)         
 Use it to specify the default editor's properties (except **onValueChanged**).
```

Comment:
```suggestion
Use it to specify default editor properties (except **onValueChanged**).
```

Reply:
I believe this edit causes ambiguity. Is it "the properties of the default editor" or "the default properties of an editor"?